tests/integration/test_web_client.py

```diff
@@ -462,3 +462,16 @@ def test_creation_lifecycle(self, antares_web: AntaresWebDesktop):
         empty_settings = StudySettings()
         new_study.update_settings(empty_settings)
         assert old_settings == new_study.get_settings()
+
+        # tests variant creation
+        parent_study = create_study_api("parent-test", "880", api_config)
```

Comment:
Last comment: It would be better to test with an existing study instead of having to create another one just for this test.
Also you could use the create_variant_api function to create a 2nd variant and perform the same tests